src/status_im2/contexts/chat/composer/style.cljs

```diff
@@ -109,3 +109,19 @@
    :blur-radius (if platform/ios? 20 10)
    :blur-type   (colors/theme-colors :light :dark)
    :blur-amount 20})
+
+(defn shell-container
+  [bottom translate-y]
+  (reanimated/apply-animations-to-style
+   {:bottom bottom} ; we use height of the input directly as bottom position
```

Comment:
maybe it would be easier to have this as:
```
[input-height translate-y]
  (reanimated/apply-animations-to-style
   {:bottom height} 
```
maybe there is less need for the comment then 🤔

Reply:
Better be clear I think. One might think why not make it `:height input-height`?

Reply:
If anything it is less clear that `bottom` here is input height if we only intend to use this for a single use case. `bottom` makes it seem re-useable but that comment will be misleading if this function is used elsewhere and not making use of input height.

imo using named variables makes this clearer.  In any case you can leave the comment but I think it's better if the code speaks for itself (with/without comments).